src/components/repository-browser.js

```diff
@@ -106,6 +95,22 @@ class RepositoryBrowser extends React.Component {
         sorter: firstBy('lastModified')
       }
     ];
+
+    if (hasUserPermission(this.props.user, permissions.DELETE_CDN_FILE)) {
+      this.columns.unshift({
```

Comment:
To me it would be nicer to have "delete" as the last column, unless I am the only one that things so :)

Reply:
It will be hidden way at the end after those horribly long columns, like this you have the name of the file and the button in the same place.

Reply:
On second thought, I like the idea of having it "less reachable" as it is a last resort admin action. So I'd also vote for last column (not very insisting though)

Reply:
No problem, done.